File: TwoSum.cpp
//https://leetcode.com/problems/two-sum/
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> sum(2);
        for( int i = 0; i < nums.size(); i++){
            for( int j = 1; j < nums.size(); j++){
                if( nums[i] + nums[j] == target ){
                    sum.at(0) = i;
                    sum.at(1) = j;
                    return sum;
                }
            }
        }
        return sum;
    }
};

File: TwoSum.py
#Python Solution intended to perform same steps as CPP solution
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for x in range(len(nums)):
            y = x + 1
            Range = len(nums)
            if y < Range:
                for y in range(y,len(nums)):
                     if nums[x] + nums[y] == target:
                            return [x,y]
